core/src/main/java/org/apache/accumulo/core/file/blockfile/cache/LruBlockCache.java

```diff
@@ -84,7 +85,7 @@
   static final int statThreadPeriod = 60;
 
   /** Concurrent map (the cache) */
-  private final ConcurrentHashMap<String,CachedBlock> map;
```

Comment:
The Lru cache factory could use the CacheType to get `"tserver.cache.data.size` or `tserver.cache.index.size`